render.yaml

```diff
@@ -36,6 +36,10 @@ services:
         fromDatabase:
           name: jaffle_db
           property: password
+      - key: PGSSLMODE
```

Comment:
This doesn't exist in the render spec: https://render.com/docs/blueprint-spec#properties-available-to-environment-variables

Reply:
ok, removed.